docs-release-notes.md

```diff
@@ -1,3 +1,31 @@
+# September 8, 2023
+
+## App Builder's Resources
+
+- Configure & Connect to an Edge Server: Fix small spelling typo.
+
+## Device Sync
+
+- Configure and Update your Data Model/Update a Data Model: Fix small spelling typo.
+- Configure Sync/Sync Settings: Update queryable fields documentation for Indexed Queryable Fields and Collection Queryable Fields.
+- Handle Errors/Sync Errors: Update queryable fields documentation for Indexed Queryable Fields and Collection Queryable Fields. 
+
+## Authenticate and MAnage Users
+
+- Manage User Sessions: Update Get a User Access Token Over HTTPS steps for clarity.
+
+## Reference
+
+- App Services Admin API: Add missing properties, ``run_as_user_id`` and ``run_as_user_id_script_source``, to the Create a New Function endpoint.
+
+## Release Notes
+
+- Update change log with release notes for v20230906.
+
+## Internal
+
+- Update the facets.csv with SubProduct value to programmatically tag the repo correctly.
```

Comment:
This one is a bit confusing because it's new to us, but there is no facets.csv in the repo and this doesn't quite capture what this PR was about. I'd probably say something like:

```suggestion
- Add programmatically-generated facet tags to the repo for improved search indexing.
```